browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1594,6 +1594,9 @@ Are you sure you want to do this?
       <message name="IDS_FINGERPRINT_LANGUAGE_SWITCH_SUMMARY" desc="Title for shields option fingerprint language">
         Reduces how much websites can learn about you based on your browser's language settings.
       </message>
+      <message name="IDS_BRAVE_SHIELDS_SAVE_CONTACT_INFO_SWITCH" desc="Shields panel toggle label that would save webcompat report contact info data when enabled.">
+        Store contact information for future webcompat reports
```

Comment:
I don't think we should use the word `webcompat` here as users might not be familiar with it. Maybe `Store contact information for reporting broken sites in the future`? cc: @rmcfadden3

Reply:
Yeah, that's fair, I updated the copy in https://github.com/brave/brave-browser/issues/40021. 

`Store contact information for future broken site reports`

`If you provide contact info it will be stored for future reports`

Reply:
Yeah, that text works for me, @ShivanKaul and @mkarolin